app/models/ip.rb

```diff
@@ -4,6 +4,14 @@ class Ip < ApplicationRecord
   validates :address, presence: true, uniqueness: true
   validate :address_must_be_valid_ip
 
+  def inactive?
+    Session
+      .where(siteIP: self.address)
+      .where("start > #{Date.today - 10.days}")
+      .limit(1)
+      .empty?
+  end
```

Comment:
This is a very naive implementation that has the potential to put a lot of load on the sessions read replica.
Before optimising this, I want to see how much load in production exactly.  Maybe it's fine as nothing else uses this database.